### PR TITLE
Track last known mouse XY

### PR DESCRIPTION
**Pull request by ruapotato**
This would solve my issue: 

"How to find X,Y position of mouse"
https://github.com/qtile/qtile/issues/1249

### CODE REVIEW

File: libqtile/core/manager.py [hunk 1]
```diff
@@ -80,6 +80,8 @@ def __init__(
 
         self._eventloop = None
         self._finalize = False
+        self.mouseX = 0
+        self.mouseY = 0
```
Comment (ramnes) [inline libqtile/core/manager.py]:
```suggestion
        self.mouse_position = (0, 0)
```

**Comment by ramnes** [comment]
@flacjacket How would you do this, roughly?

File: libqtile/core/manager.py (hunk 1 above)
Comment (ruapotato) [reply libqtile/core/manager.py]: mouse_position is a better name. :)

**Comment by flacjacket** [comment]
That's a good question. It may be that we want to pull x specific handling
out to some other location. I'm not sure how we'd hook that in, tho.

**Comment by flacjacket** [comment]
Thanks! I think we can move this functionality around when we find a better place for all of the other X11 handling stuff in the manager.

**Pull request merged**